openlibrary/macros/LoanStatus.html

```diff
@@ -126,8 +126,8 @@
       >$_("Checked Out")</a>
     </div>
 
-$elif ocaid and availability.get('is_previewable') and book_provider.short_name == 'ia':
-  $:macros.BookPreview(ocaid, analytics_attr, show_only=True)
+$elif True:
```

Comment:
@ananyakaligal it looks like there may have been a `push --force` that clobbered a lot of the changes I contributed to this branch.

Can you please use `git reflog` to see if you can identify my commit, do a `git checkout` of that commit, and then do a `git diff` to tell me what the changes were? Otherwise, it may take me a while to recover those changes which we want in the final solution :(

Reply:
Maybe some of those changes are in https://github.com/internetarchive/openlibrary/pull/10366/commits/07c60eaaf4a035183826c5260ec94aa7b08a2f19?